model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -116,17 +116,21 @@
     "!python -c \"import cv2\" || pip install -q opencv-python-headless\n",
     "\n",
     "# These are the libraries used to read DICOM Seg objects.\n",
-    "!python -m pip install -q pydicom pydicom-seg\n",
-    "\n",
-    "# Install tcia_utils to download the datasets.\n",
-    "!python -m pip install --upgrade -q --no-deps tcia_utils\n",
+    "!python -m pip install -q \"pydicom<3\" pydicom-seg\n",
     "\n",
     "# Install the dependency manually to avoid installing opencv-python.\n",
-    "!python -m pip install -q plotly bs4 ipywidgets unidecode jsonschema\n",
+    "!python -m pip install -q plotly bs4 unidecode\n",
     "!python -m pip install -q --no-deps rt-utils\n",
     "\n",
+    "# Install tcia_utils to download the datasets.\n",
+    "!python -m pip install --upgrade -q --no-deps tcia_utils\n",
+    "\n",
     "# This is the installation required for itkWidgets.\n",
-    "!python -m pip install --upgrade --pre -q \"itkwidgets[all]==1.0a23\" imjoy_elfinder"
+    "!python -m pip install --upgrade --pre -q \"itkwidgets[all]\" imjoy_elfinder\n",
```

Comment:
@kirbyju thanks for addressing this!

To clarify, we want to specify the pre-release version of itkwidgets:

```
itkwidgets[all]==1.0a53
```

but avoid/remove the `--pre` flag because it can bring in unstable transitive dependencies like `zarr==3.0.0a1`.